lms/services/group_info.py

```diff
@@ -46,9 +46,25 @@ def upsert_group_info(self, grouping: Grouping, params: dict):
         group_info.application_instance = grouping.application_instance
 
         group_info.type = self._GROUPING_TYPES[grouping.type]
-        group_info.update_from_dict(
-            params, skip_keys={"authority_provided_id", "id", "info"}
-        )
+
+        for field in [
+            # Most (all) of these are duplicated elsewhere, we'll keep updating for now
+            # because external analytics query rely on this table.
+            "context_id",
```

Comment:
The replacement here is not perfect, we still have a `setattr` but in general this model/service pair is a blast from the past.

Looking separately into what it will take to remove it.